resultstore-search-client/src/api/client/index.ts

```diff
@@ -0,0 +1,58 @@
+import * as grpcWeb from 'grpc-web';
+import { ResultStoreDownloadClient } from '../Resultstore_downloadServiceClientPb';
+import {
+    SearchInvocationsRequest,
+    SearchInvocationsResponse,
+} from '../resultstore_download_pb';
+import * as invocation_pb from '../invocation_pb';
+import { toSentenceCase } from '../../utils/index';
+
+type SetInvocations = (invocations: Array<invocation_pb.Invocation>) => void;
+type UpdateError = (error: string, hasError: boolean) => void;
+
+const destinationAddress = 'http://localhost:8080';
+const resultStore = new ResultStoreDownloadClient(
+    destinationAddress,
+    null,
+    null
+);
+const projectId = 'google.com:gchips-productivity';
```

Comment:
I recommend setting up a config file that applies this field.

Reply:
Yeah thats probably better than this. Done!